rosbridge_library/test/capabilities/test_call_service.py

```diff
@@ -50,6 +58,12 @@ def setUp(self):
             self.trigger_cb,
             callback_group=self.cb_group,
         )
+        self.trigger_long_srv = self.node.create_service(
+            Trigger,
+            self.node.get_name() + "/trigger_long",
```

Comment:
Can you just do `"trigger_long"` without the leading slash to get the same effect here?

Reply:
You mean without the node name as well? Like that
```suggestion
            "trigger_long",
```